ERROR __init__.py:97 Error: Directory migrations already exists
ERROR __init__.py:97 Error: Directory migrations already exists
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
WARNING _internal.py:87  * Debugger is active!
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 104-350-079
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 21:39:16] "GET / HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 21:40:23] "GET / HTTP/1.1" 500 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR app.py:1891 Exception on / [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2268, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask_session/sessions.py", line 166, in save_session
    time=total_seconds(app.permanent_session_lifetime))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 1472, in setex
    return self.execute_command('SETEX', name, time, value)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 775, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 789, in parse_response
    response = connection.read_response()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/connection.py", line 642, in read_response
    raise response
redis.exceptions.ResponseError: MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.
ERROR app.py:1975 Request finalizing failed with an error while handling an error
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2268, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask_session/sessions.py", line 166, in save_session
    time=total_seconds(app.permanent_session_lifetime))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 1472, in setex
    return self.execute_command('SETEX', name, time, value)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 775, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 789, in parse_response
    response = connection.read_response()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/connection.py", line 642, in read_response
    raise response
redis.exceptions.ResponseError: MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2268, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask_session/sessions.py", line 166, in save_session
    time=total_seconds(app.permanent_session_lifetime))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 1472, in setex
    return self.execute_command('SETEX', name, time, value)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 775, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 789, in parse_response
    response = connection.read_response()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/connection.py", line 642, in read_response
    raise response
redis.exceptions.ResponseError: MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 21:52:27] "GET / HTTP/1.1" 500 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR app.py:1891 Exception on / [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2268, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask_session/sessions.py", line 166, in save_session
    time=total_seconds(app.permanent_session_lifetime))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 1472, in setex
    return self.execute_command('SETEX', name, time, value)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 775, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 789, in parse_response
    response = connection.read_response()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/connection.py", line 642, in read_response
    raise response
redis.exceptions.ResponseError: MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.
ERROR app.py:1975 Request finalizing failed with an error while handling an error
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2268, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask_session/sessions.py", line 166, in save_session
    time=total_seconds(app.permanent_session_lifetime))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 1472, in setex
    return self.execute_command('SETEX', name, time, value)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 775, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 789, in parse_response
    response = connection.read_response()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/connection.py", line 642, in read_response
    raise response
redis.exceptions.ResponseError: MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2268, in process_response
    self.session_interface.save_session(self, ctx.session, response)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask_session/sessions.py", line 166, in save_session
    time=total_seconds(app.permanent_session_lifetime))
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 1472, in setex
    return self.execute_command('SETEX', name, time, value)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 775, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/client.py", line 789, in parse_response
    response = connection.read_response()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/redis/connection.py", line 642, in read_response
    raise response
redis.exceptions.ResponseError: MISCONF Redis is configured to save RDB snapshots, but it is currently not able to persist on disk. Commands that may modify the data set are disabled, because this instance is configured to report errors during writes if RDB snapshotting fails (stop-writes-on-bgsave-error option). Please check the Redis logs for details about the RDB error.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 21:54:29] "GET / HTTP/1.1" 500 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:05:03] "GET / HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:43:45] "GET / HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET / HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/css/reset.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/css/ihome/main.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/css/ihome/index.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:12] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:13] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:13] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:13] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:13] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:13] "GET /static/js/template.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:13] "GET /static/images/logo@128x59.png HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:13] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:14] "GET /static/js/ihome/index.js HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:17] "GET /api/v1.0/areas HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:17] "GET /api/v1.0/session HTTP/1.0" 200 -
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:17] "GET /api/v1.0/houses/index HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:22] "GET /register.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:22] "GET /static/css/ihome/register.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:22] "GET /static/js/ihome/register.js?v=4 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:48:23] "GET /api/v1.0/imagecode?cur=6d12ed25-b260-45aa-b38b-451303cc1af5&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:13] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:17] "GET /api/v1.0/areas HTTP/1.0" 200 -
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:18] "GET /api/v1.0/houses/index HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:19] "GET /api/v1.0/session HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:20] "GET / HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:22] "GET /api/v1.0/imagecode?cur=6d12ed25-b260-45aa-b38b-451303cc1af5&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:22] "GET /register.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:24] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:25] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:25] "GET /static/js/ihome/register.js?v=4 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:29] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:29] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:41] "GET /api/v1.0/imagecode?cur=906c99e0-9221-49cb-90f2-844941268523&pre=6d12ed25-b260-45aa-b38b-451303cc1af5 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:49:48] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:50:04] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:56:08] "GET /register.html HTTP/1.0" 304 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:56:08] "GET /api/v1.0/imagecode?cur=787c4f95-891b-4510-b06e-47c2b3689f96&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:57:00] "GET /api/v1.0/imagecode?cur=80106328-4aac-42f9-a30d-7d30949bb3b2&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 22:57:12] "GET /api/v1.0/imagecode?cur=787c4f95-891b-4510-b06e-47c2b3689f96&pre= HTTP/1.0" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:02:49] "GET /api/v1.0/session HTTP/1.0" 200 -
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:02:50] "GET /api/v1.0/houses/index HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:02:50] "GET /api/v1.0/areas HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:02:50] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.0" 404 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:02:55] "GET /login.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:02:55] "GET /static/css/ihome/login.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:02:55] "GET /static/js/ihome/login.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:03:07] "POST /api/v1.0/session HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:03:12] "GET /api/v1.0/imagecode?cur=25e614ea-2c1a-49a3-ae81-1c23f3da1a96&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:03:58] "GET /login.html HTTP/1.0" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:07:11] "GET /register.html HTTP/1.0" 304 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:07:11] "GET /api/v1.0/imagecode?cur=b4e17782-83f7-493b-94d7-9603a6216ba2&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:11] "GET /api/v1.0/imagecode?cur=b4e17782-83f7-493b-94d7-9603a6216ba2&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:20] "GET / HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET / HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/css/ihome/index.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/js/ihome/index.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/js/template.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/css/reset.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/images/logo@128x59.png HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:28] "GET /static/css/ihome/main.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:32] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff?v=4.6.3 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:32] "GET /api/v1.0/session HTTP/1.0" 200 -
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:32] "GET /api/v1.0/houses/index HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:32] "GET /api/v1.0/areas HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:33] "GET /favicon.ico HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:41] "GET /register.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:42] "GET /static/css/ihome/register.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:42] "GET /static/js/ihome/register.js?v=4 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:08:42] "GET /api/v1.0/imagecode?cur=9cdaf5f9-810d-425c-b546-a6ae76991976&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:09:25] "POST /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:13:16] "GET / HTTP/1.0" 304 -
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:13:16] "GET /api/v1.0/houses/index HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:13:16] "GET /api/v1.0/session HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:13:16] "GET /api/v1.0/areas HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:13:22] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.0" 404 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:13:28] "GET /api/v1.0/imagecode?cur=7c35b3ab-84ac-4367-aab0-690658a9faf6&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:19] "GET /register.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:19] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:19] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:19] "GET /static/css/reset.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:19] "GET /static/css/ihome/main.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:19] "GET /static/css/ihome/register.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:19] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:20] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:20] "GET /static/js/ihome/register.js?v=4 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:20] "GET /static/images/logo@128x59.png HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:21] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:21] "GET /api/v1.0/imagecode?cur=7c19ea17-b0d6-489a-930d-a6d38d4df281&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:21] "GET /static/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:28] "GET /api/v1.0/imagecode?cur=7c35b3ab-84ac-4367-aab0-690658a9faf6&pre= HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:14:58] "POST /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:15:22] "GET /api/v1.0/imagecode?cur=7c19ea17-b0d6-489a-930d-a6d38d4df281&pre= HTTP/1.0" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR app.py:1891 Exception on /api/v1.0/users [POST]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/Web/Flask_iHome3/iHome/api_1_0/passport.py", line 95, in register
    if not all([mobile, sms_code, password]):
NameError: name 'sms_code' is not defined
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:24:17] "POST /api/v1.0/users HTTP/1.0" 500 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "POST /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /index.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/css/ihome/index.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/js/template.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/js/ihome/index.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /api/v1.0/session HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /api/v1.0/areas HTTP/1.0" 200 -
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /api/v1.0/houses/index HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:38] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.0" 404 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:53] "GET /my.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:53] "GET /static/css/ihome/my.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:53] "GET /static/js/ihome/my.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:25:53] "GET /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:08] "GET /auth.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:08] "GET /static/css/ihome/profile.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:08] "GET /static/js/ihome/auth.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:08] "GET /static/js/jquery.form.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:08] "GET /api/v1.0/auth HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:28] "POST /api/v1.0/auth HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:38] "GET /index.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:39] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:40] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:41] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:41] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:42] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:42] "GET /static/js/template.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:42] "GET /static/js/ihome/index.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:44] "GET /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:49] "GET /api/v1.0/auth HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:54] "GET /my.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:54] "GET /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:56] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:57] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:58] "GET /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:26:59] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:00] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:00] "GET /static/js/ihome/my.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:08] "GET /api/v1.0/auth HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:09] "GET /profile.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:09] "GET /static/js/ihome/profile.js?v=5 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:09] "GET /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:22] "POST /api/v1.0/users/name HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:25] "GET /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:41] "GET /myhouse.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:42] "GET /static/css/ihome/myhouse.css HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:42] "GET /static/js/ihome/myhouse.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:42] "GET /api/v1.0/auth HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:42] "GET /api/v1.0/users/house HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:46] "GET /api/v1.0/users HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:53] "GET / HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:53] "GET /api/v1.0/session HTTP/1.0" 200 -
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:53] "GET /api/v1.0/houses/index HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:53] "GET /api/v1.0/areas HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:27:53] "GET /static/plugins/swiper/js/maps/swiper.jquery.min.js.map HTTP/1.0" 404 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:09] "GET /profile.html HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:11] "GET /static/js/jquery.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:12] "GET /static/js/jquery.form.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:12] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:12] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:14] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:14] "GET /static/js/ihome/profile.js?v=5 HTTP/1.0" 200 -
INFO _internal.py:122 127.0.0.1 - - [21/Jul/2019 23:28:42] "GET /api/v1.0/users/house HTTP/1.0" 200 -
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR passport.py:126 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry '17621250957' for key 'mobile'") [SQL: 'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2019, 7, 22, 18, 45, 10, 801421), datetime.datetime(2019, 7, 22, 18, 45, 10, 801448), '17621250957', 'pbkdf2:sha256:150000$LOcsBbXW$c83aeeea84d42ebf1dfbedd65f098cb12fdba58371065d0d4f4e275b22e41140', '17621250957', None, None, None)]
ERROR passport.py:126 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry '17621250957' for key 'mobile'") [SQL: 'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2019, 7, 22, 18, 45, 32, 748884), datetime.datetime(2019, 7, 22, 18, 45, 32, 748898), '17621250957', 'pbkdf2:sha256:150000$D1AzmxxW$7a77d2de11ad7328aaab36ddb2c25f171a3aea2837fb0d23c49ad9f7595a3ab2', '17621250957', None, None, None)]
ERROR passport.py:126 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry '17621250957' for key 'mobile'") [SQL: 'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2019, 7, 22, 18, 45, 34, 115330), datetime.datetime(2019, 7, 22, 18, 45, 34, 115344), '17621250957', 'pbkdf2:sha256:150000$bZghamK8$dae4a1e00e26424f27f2e63d4d40b24a68bb43c33edda3223bfb48c17b632114', '17621250957', None, None, None)]
ERROR passport.py:126 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry '17621250957' for key 'mobile'") [SQL: 'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2019, 7, 22, 18, 45, 34, 917023), datetime.datetime(2019, 7, 22, 18, 45, 34, 917049), '17621250957', 'pbkdf2:sha256:150000$El0PWZIY$96223339c679325f65f903d22e95171a4cbe1d5da044e26c58d2f947d6d02cac', '17621250957', None, None, None)]
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR passport.py:126 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry '18818225990' for key 'mobile'") [SQL: 'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2019, 7, 22, 19, 30, 7, 44078), datetime.datetime(2019, 7, 22, 19, 30, 7, 44092), '18818225990', 'pbkdf2:sha256:150000$UboqKccf$614cd91515f3dda2feb45abfee8ab508d7785066765b4cbeb746a9c20cfc7850', '18818225990', None, None, None)]
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR profile.py:183 "Please check your BUCKET_NAME! The UpHosts is {'error': 'accessKey is not found'}"
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:46 结束日期必须大于开始时期
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:132 Invalid input of type: 'dict'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR passport.py:126 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry '13383890993' for key 'mobile'") [SQL: 'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2021, 4, 27, 11, 49, 10, 986626), datetime.datetime(2021, 4, 27, 11, 49, 10, 986645), '13383890993', 'pbkdf2:sha256:150000$Yg5Dtp8W$d9de42f4deafd1043449cc47266544c42bfb11961809a28cd986074f29e43e4d', '13383890993', None, None, None)]
ERROR passport.py:126 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry '13383890993' for key 'mobile'") [SQL: 'INSERT INTO ih_user_profile (create_time, update_time, name, password_hash, mobile, real_name, id_card, avatar_url) VALUES (%s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (datetime.datetime(2021, 4, 27, 11, 50, 1, 37182), datetime.datetime(2021, 4, 27, 11, 50, 1, 37196), '13383890993', 'pbkdf2:sha256:150000$6lXbwPAB$31ef42604487decd3445c8f5622dcc89502c405e16cd618f7bddd2ef39c8c4f8', '13383890993', None, None, None)]
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR profile.py:183 "Please check your BUCKET_NAME! The UpHosts is {'error': 'accessKey is not found'}"
ERROR profile.py:147 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry 'Sunshine' for key 'name'") [SQL: 'UPDATE ih_user_profile SET update_time=%s, name=%s WHERE ih_user_profile.id = %s'] [parameters: (datetime.datetime(2021, 4, 27, 11, 51, 47, 773711), 'Sunshine', 2)]
ERROR profile.py:147 (MySQLdb._exceptions.IntegrityError) (1062, "Duplicate entry 'Sunshine' for key 'name'") [SQL: 'UPDATE ih_user_profile SET update_time=%s, name=%s WHERE ih_user_profile.id = %s'] [parameters: (datetime.datetime(2021, 4, 27, 11, 51, 51, 800151), 'Sunshine', 2)]
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:421 Invalid input of type: 'list'. Convert to a byte, string or number first.
ERROR house.py:173 Invalid input of type: 'list'. Convert to a byte, string or number first.
